Druid/DRScope.class.st

```diff
@@ -2,9 +2,164 @@ Class {
 	#name : 'DRScope',
 	#superclass : 'Object',
 	#instVars : [
-		'outerScope'
+		'argumentNames',
+		'numArgs',
+		'tempVars',
+		'copiedVars',
+		'tempVector',
+		'id',
+		'tempVectorVar',
+		'node',
+		'outerScope',
+		'opalScope'
```

Comment:
Ok, we are still using the opalScope because of the builder. But we cannot generate it in a method?